osc/osc.go

```diff
@@ -526,6 +508,11 @@ func (c *Client) Send(packet Packet) error {
 // Server
 ////
 
+var (
+	initBuf = make([]byte, 65535)
+	buf  = bytes.NewBuffer(initBuf)
```

Comment:
This should probably be initialized per `server` to avoid a race condition (two `server`'s running at the same time would be using the same `buf`). But I'm curious what everyone else thinks is the best way about this.